src/lib/CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ find_package(
 include(UseSWIG)
 find_package(pybind11 REQUIRED)
 
+if(UNIX AND NOT APPLE)
```

Comment:
In my opinion embarrassing that Apple ships its dev tools with OpenMP disabled by default... it is possible to install it with `brew` and compile it, but as you noted a task for another time if the need arises